docs/source/science/properties/morphology.rst

```diff
@@ -3,4 +3,97 @@
+----------------------------------------------------------------
+
+Generating atomistic morphologies [1]_, [2]_, [3]_, [4]_ is a critical starting point for understanding and predicting the properties of organic semiconductors. By generating atomistic morphologies, one can obtain secondary properties such as charge carrier mobility, conductivity, optical absorption spectra, ionization potentials, and electron affinities of the embedded molecules. These properties are crucial for the comprehensive understanding and optimization of materials for specific applications.
+
+
+Nanomatch Multiscale Workflow for Morphology
+--------------------------------------------
+
+In the DiaDEM Platform, the Nanomatch Multiscale Workflow for morphology is employed (:ref:`science_calculators_morphology`), which includes:
+
+- Molecule parametrization,
+- Morphology deposition using a Monte-Carlo protocol mimicking physical deposition (*DEPOSIT protocol* [1]_),
+- Analysis of the generated morphology using various computational tools,
+- Calculation of key morphological properties such as radial distribution function, mass and number density, mean distance to the nearest neighbours.
+
+The morphology can be downloaded under the name `structure.cml` along with other files (:ref:`science_calculators_morphology_files`).
+
+The morphological properties, which are computed within the calculator  :ref:`science_calculators_morphology` are explained in the Table below.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+     - Units
+     - Value (example)
+     - Additional Information
+   * - average_neighbors
+     - Average distance to neighboring molecules
+     - Å
+     - 17.6
+     - Average distance to neighboring molecules
+   * - mass_density
+     - Mass per unit volume
+     - g/cm³
+     - 1.14
+     - Standard deviation (`std`): 0.01 (example value)
+   * - molecular_volume
+     - Volume occupied by a single molecule
+     - nm³
+     - 0.23
+     -
+   * - number_density
+     - Number of molecules per unit volume
+     - 1/cm³
+     - 4.36e+21
+     - Standard deviation (`std`): 9.9e+19 (example value)
+   * - rdf_first_peak
+     - Position of the first peak in the radial distribution function
+     - Å
+     - 4.921630094043887
+     - Indicates the most probable intermolecular distance
+
+
+Using existing or generated morphology, the determination of secondary properties that depend on the morphology, such as charge carrier mobility, ionization potentials (IP), and electron affinities (EA) as possible as well.
+For example, see :ref:`science_properties_mobility`.
+
+Although this workflow integrates extensive experience and advanced simulation methods, including quantum chemistry (QM), molecular mechanics (MM), Monte-Carlo, the only input required from the user is the structure of the molecule, for example, InChI or another identifier. This makes it accessible to all users, regardless of their background.
```

Comment:
That does not matter for DiaDEM - the user just clicks the button. I'd remove this sentence but I also don't mind if you want to keep it ;-)

Reply:
This is kind of marketing to my opinion.

I made it less technichal:

Although this workflow integrates extensive experience and advanced simulation methods, including quantum chemistry (QM), molecular mechanics (MM), and Monte Carlo simulations, the user only needs to know the structure of the molecule. This makes the workflow accessible to all users, regardless of their background.

This statement is general for almost all out calculators, except for those that are simply based on one QC software...